include/tunnler/room.h

```diff
@@ -82,6 +82,13 @@ class Room final {
     void HandleJoinRequest(const RakNet::Packet* packet);
 
     /**
+     * Adds the senders nickname to a chat message and broadcasts this packet
```

Comment:
should just be "sender" or "sender information" or something - so if we change this to just send a member index or something we don't have to update it here